src/main/java/com/google/api/codegen/ApiConfig.java

```diff
@@ -61,6 +61,14 @@ public static ApiConfig createApiConfig(Model model, ConfigProto configProto) {
     }
   }
 
+  /**
+   * Creates an ApiConfig with no content. Exposed for testing.
+   */
+  public static ApiConfig createDummyApiConfig() {
```

Comment:
It looks like this is only used in order to extract the snippet file names from GapicProviderFactory. I wonder if it would be better expose the snippet file names directly, without exposing a test-only function in non-test code.